index.html

```diff
@@ -214,6 +214,24 @@ <h2 class="title-about">About<br>Myself</h2>
       </svg></div> 
       <hr width="100%" color="#DFE1E6" size="2px">     
   </section>
+  <footer class="contact-form">
+    <div class="first-container" >
+         <h2 class="contact-head">Contact me </h2>
+        <p class="para-contact">If you have an application you <br> are interested in developing, a <br>  feature that you need built or a <br> project that needs coding. <br> I’d love to help with it </p>
+    </div>
+    <form class="contact-list" action="https://formspree.io/f/xdobrbok" method="post">
+             <input type = "text" id="name"  name="user_name" maxlength = "30" placeholder="Enter Your name"  required>
+            <input type="email" id="email" name="user_email" placeholder="Enter your email" required>
+            <textarea name="message" id="msg" cols="24" rows="6" maxlength = "500" placeholder="Write your message here" required></textarea>
+            <button type="submit" class="btn">Get It Touch</button>
```

Comment:
[OPTIONAL] Kindly note that your button's color is different from the figma design. Its good practice to have the design match figma desin
This is your buttons color
<img width="152" alt="Screenshot 2022-09-28 at 12 39 05" src="https://user-images.githubusercontent.com/48546992/192746311-803635e9-bb33-42b5-8093-1b64a7b8eb34.png">

This is the color in the figma design
<img width="171" alt="Screenshot 2022-09-28 at 12 43 00" src="https://user-images.githubusercontent.com/48546992/192746829-8df9a8b4-bf97-4e7d-bd6a-5097040609ba.png">